articles/azure-functions/functions-monitor-log-analytics.md

```diff
@@ -21,11 +21,11 @@ Azure Monitor uses a version of the [Kusto query language](/azure/kusto/query/)
 
 ## Setting up
 
-From the Monitoring section, select **Diagnostic settings** and then click **Add**.
+From the **Monitoring** section of your function's App Service, select **Diagnostic settings** and then click **Add diagnostic setting**.
```

Comment:
Functions is not part of app service.
Please change the wording to remove app service mention to
From the Monitoring section, select **Diagnostic settings** and then click **Add diagnostic setting**.